### PR TITLE
=str minor typo fixes and scala version bump

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

**Comment by 2m** [comment]
PLS BUILD

**Comment by drewhk** [comment]
LGTM

File: akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala [hunk 1]
```diff
@@ -15,7 +15,7 @@ import scala.collection.mutable
 private[akka] object StreamLayout {
 
   // compile-time constant
-  val debug = true
+  final val debug = true
```
Comment (ktoso) [inline akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala]: +1 (I'd `Debug`)

**Comment by ktoso** [comment]
LGTM

**Comment by rkuhn** [comment]
LGTM

**Comment by rkuhn** [comment]
merge?

**Pull request merged**